@remirror/showcase/tsconfig.prod.json

```diff
@@ -5,7 +5,8 @@
     "baseUrl": "src",
     "paths": {},
     "composite": true,
-    "emitDeclarationOnly": true,
+    "emitDeclarationOnly": false,
+    "module": "commonjs",
```

Comment:
:rotating_light: DON'T MERGE THIS

I don't know why this seems required; I think I need catching up on the build system (and once caught up I can write up some instructions for others).

Reply:
Sure thing, I'll hold.